content/markdown/texas-gis-forum/2018/index.md

```diff
@@ -13,7 +13,6 @@ mainimagesm: https://cdn.tnris.org/images/largest_tree_graphic_sm.jpg
 Take a look at the  <a href="/texas-gis-forum/2018/agenda">agenda</a> to see available speaker presentations:<br>
   <a class="btn btn-lg btn-tnris" href="/texas-gis-forum/2018/agenda">View the Agenda</a></p>
 
-<p class="lead">Revisit the forum with our photo recap:<br>
   <h3>Platinum Sponsors</h3>
```

Comment:
where's the closing </p> tag? the html in the markdown is very difficult to read so we just need to be careful here. one missing tag could potentially cause the prod build to fail.

Reply:
I'm guessing there used to be a button or link to view a photo recap here, and maybe when that was removed the closing `</p>` tag got deleted with it. I initially deleted this part since I didn't see any link to the resource it was referring to. 

I looked back through this file and there was some wonky stuff going on with `<p>` tags all over it... closing tags before any opening tags, open tags that never got closed, tags used in places with no text. My latest push should solve all of these.